package.json

```diff
@@ -64,7 +64,6 @@
     "vite-plugin-environment": "^1.1.3",
     "vitest": "^0.33.0",
     "vue-class-component": "^7.2.6",
-    "vue-plugin-helper-decorator": "^0.0.12",
```

Comment:
Darn, I tried this locally and it looks like SB need this anyway:
```
"vuelidate-property-decorators": "1.0.28",
```

Reply:
Are you positive Sev? I was able to see BaseAddress on SB even when I removed this and rebuilt.

Reply:
![baseaddress](https://github.com/bcgov/bcrs-shared-components/assets/122301442/27aa6682-f6da-4651-8634-e75e1eca5677)

Reply:
^^ Yeah but there's no schema, and that's what causes the validation to fail.

Reply:
You fixed this with an older version of Vuelidate so this is resolved.